src/app/core/auth/eua.interceptor.ts

```diff
@@ -0,0 +1,11 @@
+import { AuthInterceptor } from './auth.interceptor';
+
+export class EuaAuthInterceptor extends AuthInterceptor {
```

Comment:
@sshah-asymmetrik do these subclasses need the `@Injectable` annotation, or does the parent class cover that?

Reply:
@brianghig They seem to work without the annotation, which makes me think they're inheriting it from the parent class. That being said, I think it's better to include the annotation anyways. It doesn't do any harm and can serve as preparation for if the interceptor were changed to be a standalone that doesn't extend anything.